api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -313,6 +314,15 @@ private static void setResearchPurposeDetails(org.pmiops.workbench.db.model.Work
     dbWorkspace.setPopulation(purpose.getPopulation());
     dbWorkspace.setPopulationOfFocus(purpose.getPopulationOfFocus());
     dbWorkspace.setAdditionalNotes(purpose.getAdditionalNotes());
+    dbWorkspace.setContainsUnderservedPopulation(purpose.getContainsUnderservedPopulation());
```

Comment:
Do you need to modify TO_CONTROLLER_WORKSPACE? (If you don't, we'll never return this information to the client...)

OTOH, I don't know if you want to fetch this information for every workspace in a list; as written that would be O(n) queries. So you might just want to populate these fields on the methods that return individual workspaces?